backend/migrator/migration/prod/2.20/0001##review_config.sql

```diff
@@ -0,0 +1,46 @@
+-- Create review config table.
+CREATE TABLE review_config
+(
+    id BIGSERIAL PRIMARY KEY,
```

Comment:
Was thinking introducing resource-id type of id (string) instead of serial.

Declarative definition for terraform

Reply:
String type resource-id also LGTM. How do we generate the resource id during the migration, maybe just use `CONCAT('review-config-', policy.id::text)` as the resource id?

Reply:
How about the environment id such as environment-prod.